src/sonic-yang-models/yang-models/sonic-grpcclient.yang

```diff
@@ -0,0 +1,96 @@
+module sonic-grpcclient {
+    namespace "http://github.com/sonic-net/sonic-grpcclient";
+    prefix grpcclient;
+    yang-version 1.1;
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONiC DualToR grpc client configuration data";
+
+    revision 2024-10-14 {
+        description
+            "Initial revision";
+    }
+
+    container sonic-grpcclient {
+
+        container GRPCCLIENT {
+
+            container CONFIG {
```

Comment:
This is case sensitive. Need to put them lowercase to validate yang. otherwise will fail:
```
sonic_yang(3):All Keys are not parsed in GRPCCLIENT
dict_keys(['certs', 'config'])
sonic_yang(3):exceptionList:[]
sonic_yang(3):Data Loading Failed:All Keys are not parsed in GRPCCLIENT
dict_keys(['certs', 'config'])
```

Reply:
fixed